### PR TITLE
Signup Page

### PR DESCRIPTION
**Pull request by ameentalahmeh**
- Call the `Main Form` component, and passing `signup details` to it.

### CODE REVIEW

File: student/src/components/Form/index.style.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ export const StyledForm = styled(Form)`
 
 export const StyledLabel = styled.label`
   width: 100%;
+  margin-bottom: 15px;
+  font-size: 14px;
```
Comment (mhmdtshref) [inline student/src/components/Form/index.style.js]: Make the `font-size` related to the width

File: student/src/components/Pages/Signup/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            { label:"What is your gender ?", type:"radio", options: GenderOptions},
+            { label:"Is English your first language ?", type:"radio", options: EnglishOptions},
+            { type: "select", component: "select", name: "ageranges", options: ageOptions},
+            { type: "text", name:"postalcode", placeholder: "Postal Code"},
+              ]
+        return (
+          <React.Fragment >
+          <StyledPage>
+          <StyledBk src={bg}/>
+          <StyledTitle> Enter your information </StyledTitle>
```
Comment (mhmdtshref) [inline student/src/components/Pages/Signup/index.js]: Reformat the HTML style

**Review by mhmdtshref — CHANGES_REQUESTED** [review]
Run `eslint:fix` and view my comments

**Review by RazanTayyem — APPROVED** [review]
Done :+1:

File: student/src/components/Pages/Signup/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            {label: "14", value:"14"},
+            {label: "15", value:"15"},
+            {label: "16", value:"16"},
+            {label: "17", value:"17"},
+            {label: "18", value:"18"},
+              ];
+      const fields = [
+            { type: "text", name:"username", placeholder: "Enter username"},
+            { type: "text", name:"password", placeholder: "Enter password"},
+            { label:"What is your gender ?", type:"radio", options: GenderOptions},
```
Comment (ghassanmas) [inline student/src/components/Pages/Signup/index.js:37]: Change the type for password field to be `password` rather than `text`

**Review by ghassanmas — APPROVED** [review]

File: student/src/components/Pages/Signup/index.js (hunk 3 above)
Comment (ameentalahmeh) [reply student/src/components/Pages/Signup/index.js:37]: done :+1:

**Pull request merged**